src/rcamera.h

```diff
@@ -150,15 +150,15 @@ void SetCameraMoveControls(int keyFront, int keyBack,
 #endif
 
 // Camera mouse movement sensitivity
-#define CAMERA_MOUSE_MOVE_SENSITIVITY                   0.5f    // TODO: it should be independant of framerate
+#define CAMERA_MOUSE_MOVE_SENSITIVITY                   0.5f    // TODO: it should be independent of framerate
 #define CAMERA_MOUSE_SCROLL_SENSITIVITY                 1.5f
 
 // FREE_CAMERA
 #define CAMERA_FREE_MOUSE_SENSITIVITY                   0.01f
 #define CAMERA_FREE_DISTANCE_MIN_CLAMP                  0.3f
 #define CAMERA_FREE_DISTANCE_MAX_CLAMP                  120.0f
 #define CAMERA_FREE_MIN_CLAMP                           85.0f
-#define CAMERA_FREE_MAX_CLAMP                          -85.0f
+#define CAMERA_FREE_MAX_CLAMP                          (-85.0f)
```

Comment:
The use of parentheses on the negative value helps avoid errors that arise depending on where CAMERA_FREE_MAX_CLAMP might be inserted.  It would be a mistake; it might go undetected.  Of course that could be the case for any use of a preprocessor definition.   I don't disagree with Ramon's preference.